query-coordinator/src/main/scala/com/socrata/querycoordinator/fusion/CompoundTypeFuser.scala

```diff
@@ -236,7 +236,7 @@ class CompoundTypeFuser(fuseBase: Map[String, String]) extends SoQLRewrite {
                   .map(subProp => ColumnOrAliasRef(NoQualifier, ColumnName(s"${name.name}_$subProp"))(NoPosition))
                 Some(FunctionCall(SoQLFunctions.HumanAddress.name, args, filter, window)(NoPosition, NoPosition))
               case _ =>
-                throw BadParse("unknown location property", fc.position)
```

Comment:
For context, errors got refactored in soql-reference so `BadParse` was no longer exposed. It was always the wrong exception to throw, however. What's happening here is that we've parsed the data successfully but doesn't have components we expect.